### DISCUSSION TITLE
multiple `<clef>` elements in single `<attribute>`?

### DISCUSSION THREAD

**Discussion by lemzwerg** [w3c-cg/musicxml#508] [General]
Looking into the MusicXML 4.0 specification I see that almost all children of `<attribute>` may occur more than once.  Why?  For example, what would it mean if I have two `<clef>` children?  Testing with Finale and MuseScore I see that only the last of those children gets used (which is sensible IMHO).

I could imagine that having multiple elements makes sense if the final value is the accumulative of all elements (i.e., defining some properties in element  A and some others in element  B), but this is definitely not the case...

AFAICS, there is no clarification in MusicXML of such situations.

**Comment by mscuthbert** [comment]
A part can have multiple staves. The number attribute of clef and other objects defines which staff the clef belongs to.

**Comment by lemzwerg** [comment]
Ah, ok, thanks!  Didn't think of that.

Is there a (complete) example MusicXML file somewhere that demonstrates this?

**Comment by mdgood** [comment]
Besides lots of examples in the clef element documentation, this page has many larger examples: https://www.musicxml.com/music-in-musicxml/example-set/

**Comment by lemzwerg** [comment]
Thanks again.

For other people who are reading this, the MusicXML standard example for `<part-symbol>` shows a use case.
